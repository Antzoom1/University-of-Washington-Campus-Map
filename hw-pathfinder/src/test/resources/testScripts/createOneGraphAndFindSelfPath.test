# Find a path with Dijkstra's algorithm between
# the same node.

CreateGraph simpleGraph

AddNode simpleGraph B

FindPath simpleGraph B B