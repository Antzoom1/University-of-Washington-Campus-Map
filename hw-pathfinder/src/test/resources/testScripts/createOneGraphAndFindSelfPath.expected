# Find a path with Dijkstra's algorithm between
# the same node.

created graph simpleGraph

added node B to simpleGraph

path from B to B:
total cost: 0.000